dmensamenu/dmensamenu.py

```diff
@@ -104,7 +108,11 @@ def main():
     else:
         canteen = args.id[0]
     if canteen is not None and not args.search:
-        meals = getmenu(canteen)
+        day = None
+        if args.forecast and time.localtime().tm_hour > 14:
+            day = datetime.date.today() + datetime.timedelta(days=1)
```

Comment:
We should show the menu of the next day the canteen is open instead of tomorrow, since this could for example be a Sunday.